Formula/fail2ban.rb

```diff
@@ -98,8 +100,8 @@ def install
     inreplace "setup.py", "platform_system in ('linux',", "platform_system in ('linux', 'darwin',"
 
     system "./fail2ban-2to3"
-    system "python3", *Language::Python.setup_install_args(libexec),
-                      "--install-lib=#{libexec/Language::Python.site_packages("python3")}",
+    system python3, *Language::Python.setup_install_args(libexec),
+                      "--install-lib=#{libexec/site_packages}",
                       "--without-tests"
```

Comment:
```suggestion
    system python3, *Language::Python.setup_install_args(libexec, python3), "--without-tests"
```
`setup_install_args` sets `--install-lib` correctly now if you pass the path to `python3` as its second argument. (In general we probably always want to pass the second argument.)

Reply:
thanks!